tests/unit/adapters/drf-test.js

```diff
@@ -33,6 +33,7 @@ test('buildURL', function(assert) {
   var adapter = this.subject();
   assert.equal(adapter.buildURL('Animal', 5, null), 'test-host/test-api/animals/5/');
   assert.equal(adapter.buildURL('FurryAnimals', 5, null), 'test-host/test-api/furry-animals/5/');
+  assert.equal(adapter.buildURL('Animal', null, null, 'query', { limit: 10 }), 'test-host/test-api/animals/');
```

Comment:
Not sure if related, but latest ember data added the queryRecord method to the rest adapter class

https://github.com/emberjs/data/releases/tag/v1.13.7